swift/ql/test/library-tests/dataflow/taint/libraries/url.swift

```diff
@@ -287,7 +287,7 @@ func taintThroughURL() {
 	let _ = clean.withCString({
 		ptrClean in
 		sink(arg: URL(fileURLWithFileSystemRepresentation: ptrClean, isDirectory: false, relativeTo: nil))
-		sink(arg: URL(fileURLWithFileSystemRepresentation: ptrClean, isDirectory: false, relativeTo: urlTainted)) // $ MISSING: tainted=210
+		sink(arg: URL(fileURLWithFileSystemRepresentation: ptrClean, isDirectory: false, relativeTo: urlTainted)) // $ tainted=210
```

Comment:
This is inside a call to `String.withCString` and references `urlTainted` from outside, so it plausibly affected by one of the closure captures changes.  The model for `withCString` was created recently as well, though in this case I don't see why it matters.

Ah well, it's a positive result!